src/main/java/qla/modules/confuguration/AppConfiguration.java

```diff
@@ -14,8 +14,7 @@
 
 public class AppConfiguration {
 	private static Properties properties;
-	private static String CONFIG_FOLDER = System.getProperty("config.folder");
-	private static String SETTINGS_FILE = CONFIG_FOLDER + File.separator+"config.properties";
+	private static String CONFIG_FOLDER = System.getProperty("config.folder")==null?"D:\\installed\\logAnalyzer\\config":System.getProperty("config.folder");	private static String SETTINGS_FILE = CONFIG_FOLDER + File.separator+"config.properties";
```

Comment:
Need to remove hardcode ;))

Reply:
Makes sense) 
It was added only for test purposes :)